cli/operator/register.go

```diff
@@ -136,21 +136,38 @@ func RegisterCmd(p prompter.Prompter) *cobra.Command {
 				llog,
 				noopMetrics,
 			)
+			if err != nil {
+				return err
+			}
 
+			reader, err := elContracts.NewELChainReader(
+				elContractsClient,
+				llog,
+				ethClient,
+			)
 			if err != nil {
 				return err
 			}
 
-			_, err = elWriter.RegisterAsOperator(ctx, operatorCfg.Operator)
+			status, err := reader.IsOperatorRegistered(context.Background(), operatorCfg.Operator)
 			if err != nil {
 				return err
 			}
 
+			if !status {
+				_, err = elWriter.RegisterAsOperator(ctx, operatorCfg.Operator)
+				if err != nil {
+					return err
+				}
+			} else {
+				llog.Info("Operator is already registered")
+			}
+
 			_, err = elWriter.RegisterBLSPublicKey(ctx, keyPair, operatorCfg.Operator)
 			if err != nil {
 				return err
 			}
-			fmt.Println("Operator registered successfully")
+			llog.Info("Operator is registered and bls key added successfully")
```

Comment:
is llog a typo?

Reply:
I think I did it coz i wanted to do `log` and there was a native `log` so I added `llog`. I can do logger.

Reply:
fixed